frontend/package.json

```diff
@@ -122,14 +122,14 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.9.0",
     "@patternfly/patternfly": "4.122.2",
+    "@patternfly/quickstarts": "1.0.0",
```

Comment:
https://github.com/openshift/console/pull/9550/files#diff-da6498268e99511d9ba0df3c13e439d10556a812881c9d03955b2ef7c6c1c655R125 will bump this to `1.0.1`